server/cluster.go

```diff
@@ -172,6 +189,13 @@ func makeRegionSearchKey(clusterRootPath string, endKey []byte) string {
 	return strings.Join([]string{clusterRootPath, "k", encodeRegionSearchKey(endKey)}, "/")
 }
 
+func makeJobKey(clusterRootPath string, jobID uint64) string {
+	// We must guarantee the job handling order, so use %020d to format the job key,
+	// use etcd range get to get the first job and then handle it.
+	// Should we use a 8 bytes binary BigEndian instead of 20 bytes string?
+	return strings.Join([]string{clusterRootPath, "j", fmt.Sprintf("%020d", jobID)}, "/")
```

Comment:
I think j is ok, I want to use a single char for different types. And if we want to use detailed flag, I think job is better.